testsuite/gnat2goto/testsuite.py

```diff
@@ -74,19 +74,26 @@ def collect_result_including_uok(name, process, _job_info):
             test_status = test_result[0].split(':')[0]
             if test_status == 'UOK':
                 test_metrics['uok'] += 1
+            elif test_status == 'INVALID_TEST':
+                test_metrics['invalid'] += 1
 
     run_testcase = generate_run_testcase('run-test', discs, m.options)
 
     MainLoop(test_list, run_testcase, collect_result_including_uok, m.options.mainloop_jobs)
 
     print "Summary: Ran %(run)s/%(total)s tests, with %(failed)s failed, %(crashed)s crashed, %(uok)s unexpectedly passed." % test_metrics
+    if test_metrics['invalid'] > 0:
+        print 'Some tests are invalid'
 
     # Generate the report file
     ReportDiff(m.options.output_dir,
                m.options.old_output_dir).txt_image(m.options.report_file)
 
-    if test_metrics['failed'] > 0 or test_metrics['crashed'] > 0 or test_metrics['uok'] > 0:
-        sys.exit(1)
+    if test_metrics['failed'] > 0 or \
```

Comment:
Do you actually need the backslashes here?

Reply:
it didn't work without them, but honestly my python is kind of weak so maybe there's another way?

Reply:
https://www.python.org/dev/peps/pep-0008/#multiline-if-statements maybe?